### PR TITLE
HKISD-159 add script to add eri kaupunginosia location

### PR DESCRIPTION
**Pull request by millaeerola**
- add a script to create "Eri kaupunginosia" division under every district
- will be removed once divisions are created

### CODE REVIEW

File: sonar-project.properties
Comment (markspl) [inline sonar-project.properties:1]: Nitpicking 😅 the row includes two commas next to each other.

File: infraohjelmointi_api/management/commands/locationimporter.py [hunk 1]
```diff
@@ -17,16 +20,24 @@ def add_arguments(self, parser):
             ),
             default="",
         )
+        parser.add_argument(
+            "--eri-kaupunginosia",
```
Comment (markspl) [inline infraohjelmointi_api/management/commands/locationimporter.py:24]: I was wondering, should we keep script arguments in English. 🤔

Comment (millaeerola) [reply infraohjelmointi_api/management/commands/locationimporter.py:24]: Generally speaking, yes. But in this case the location option is called "Eri kaupunginosia" so I think it makes sense and is easier to connect to the option shown in the UI. Also, this script will be deleted after the change is done. It's used only once to add the new option to the database

**Review by markspl — APPROVED** [review]
🚀

**Pull request merged**